Library/Homebrew/sandbox.rb

```diff
@@ -141,8 +141,15 @@ def path_filter(path, type)
   class SandboxProfile
     SEATBELT_ERB = <<-EOS.undent
       (version 1)
+      (deny network-inbound)
+      (deny network-outbound)
       (debug deny) ; log all denied operations to /var/log/system.log
       <%= rules.join("\n") %>
+      (allow network-outbound
+        (subpath "/tmp")
+        (subpath "/private")
+        (subpath "#{HOMEBREW_TEMP}")
```

Comment:
Ok, thanks. Any thoughts on this general approach and whether it will make sense? Just wondering why you decided not to do this (or blocking any file reads) before?